app/presenters/track.rb

```diff
@@ -17,21 +17,21 @@ def method_missing(method, *args)
       def docs
         track_docs = trackler_track.docs("/api/v1/tracks/%s/images/docs/img" % @track_id)
 
-        track_docs.each_pair do |topic_name, topic_content|
-          track_docs[topic_name] = if topic_content.present?
-                                     topic_content
-                                   else
-                                     fallback_topic_content(topic_name)
-                                   end
+        track_docs.each_pair do |topic_key, topic_content|
+          track_docs[topic_key] = if topic_content.present?
+                                    [topic_content.strip, better_content(topic_key)].join("\n")
+                                  else
+                                    fallback_topic_content(topic_key)
+                                  end
```

Comment:
I totally agree. I took a stab at breaking the `docs` method apart in https://github.com/exercism/exercism.io/pull/3475/commits/1a44a1e9248458d1ef1803e93a7a2fa112ad9fd1. What do you think?